README.md

```diff
@@ -19,119 +19,87 @@
 
-Make a duplicate of the `config/vectara-website-search/` directory and rename it `pg-search/`.
+`npm run bootstrap`
 
-Update the `config/pg-search/config.yaml` file with these changes:
+This one-liner installs dependencies, runs the configuration script, and spins up the local application. **When prompted for which application to create, simply select one of the three default apps, and you're good to go!** We'll work on setting up a custom application later in this doc.
 
-- Change the **corpus_id** value to the ID of the corpus into which you ingested Paul Graham's essays as part of the [`vectara-ingest` Quickstart](https://github.com/vectara/vectara-ingest/blob/main/README.md#quickstart).
-- Change the **account_id** value to the ID of your account. You can click on your username in the top-right corner to copy it to your clipboard.
-- Change the **app_title** to "Ask Paul Graham".
+If you would like to run the setup steps individually, you can run:
 
-Edit the `config/pg-search/queries.json` file and update the four questions to a set of curated questions you'd like to include in the user interface. For example: "What is a maker schedule?"
+- `npm install`: for installing dependencies
+- `npm run configure`: for running the configuration script
+- `npm run start`: for running the application locally
 
-### 4. Run the application
+Congratulations! You've just setup and run a sample app powered by Vectara!
 
-Execute the run script from the root directory using your `config/` directory, assigning the **default** profile from your secrets file:
+### Deployment
 
-```sh
-bash docker/run.sh config/pg-search default
-```
+To set up a deployable docker image, do the following:
 
-The application executes inside of a [Docker container](https://www.docker.com/resources/what-container/) to avoid any issues with existing environments and installed packages.
+- Install [Docker](https://docs.docker.com/engine/install/). Then make sure it's running on your machine.
+- Run `bash docker/run.sh`
+- View app logs with `docker logs -f vanswer`
+- Stop the app with `docker stop vanswer`
 
-When the container is set up, the `run.sh` launch script will open up a browser at `localhost:80`.
+`run.sh` creates a deployable docker image and runs it. The app should be accessible at `http://localhost:80`.
 
-### 5. Done!
+## Building Your Own Application
 
-Your application is now up and running. Try a few queries to see how it works.
+### Prerequisites
 
-You can view your application's logs by running `docker logs -f vanswer`. You can stop the application and shut down the container with `docker stop vanswer`.
+When building your own application, you will need to:
 
-## Project architecture
+- [Create a corpus](https://docs.vectara.com/docs/console-ui/creating-a-corpus) via the Vectara Console
+- [Ingest data](https://github.com/vectara/vectara-ingest/blob/main/README.md#quickstart) into the corpus you just created
```

Comment:
Let's simplify the language to try to use more accessible terms, and also clarify that there isn't a hard dependency on Vectara Ingest. Here's what I suggest:

---

- **Create a data store.** Log into [Vectara Console](https://console.vectara.com/) and [create a data store](https://docs.vectara.com/docs/console-ui/creating-a-corpus).
- **Add data to the data store.** You can use [Vectara Ingest](https://github.com/vectara/vectara-ingest) to crawl datasets and websites, or use our [Indexing APIs](https://docs.vectara.com/docs/api-reference/indexing-apis/indexing) directly.

Reply:
https://github.com/vectara/vectara-answer/pull/99/commits/25ddc03a88a276af279e5b0b277f33abb0395eec